src/libs/runtime-utils.js

```diff
@@ -235,17 +235,19 @@ export const appIsSettingUp = app => {
   return app && (app.status === 'PROVISIONING' || app.status === 'PRECREATING')
 }
 
```

Comment:
I will make the first 2 suggested changes, and I am going to rename "getCurrentAppForType" to just "getCurrentApp". All the methods now take appType as an argument, and the original (temporarily deprecated) getCurrentApp that assumed Galaxy is gone, so I think it is best to drop the "ForType" in the name.

For the 3rd suggestion, I would prefer not to touch that, especially with the lack of tests to make sure that I'm actually implementing the right behavior for the different statuses. But the currentAppIncludingDeleting method is only used within runtime-utils itself, so I will change it so that is not exported (and it is renamed getCurrentAppIncludingDeleting).

Reply:
I wrote a unit test and made the third proposed change as well.